src/main/java/com/minecolonies/coremod/util/StructureWrapper.java

```diff
@@ -284,6 +279,8 @@ public boolean doesStructureBlockEqualWorldBlock()
     {
         final IBlockState structureBlockState = structure.getBlockState(this.getLocalPosition());
         final Block structureBlock = structureBlockState.getBlock();
+        final Template.EntityInfo entityInfo = structure.getEntityinfo(this.getLocalPosition());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the declaration of "entityInfo" closer to the code that uses it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1941)